akka-stream/src/main/scala/akka/stream/impl/fusing/ActorInterpreter.scala

```diff
@@ -142,18 +144,25 @@ private[akka] class BatchingActorInputBoundary(val size: Int)
     case OnSubscribe(subscription) ⇒ subscription.cancel() // spec rule 2.5
   }
 
-  private def completed: Actor.Receive = {
-    case OnSubscribe(subscription) ⇒ throw new IllegalStateException("Cannot subscribe shutdown subscriber")
-  }
+}
 
+private[akka] object ActorOutputBoundary {
+  /**
+   * INTERNAL API.
+   */
+  private[akka] case object ContinuePulling extends DeadLetterSuppression
```

Comment:
right; I guess I was sloppy because it doesn’t really matter anyway—I’m still disappointed that for the JVM all of these choices mean `public` :-(